scripts/upload_translatable_strings.bash

```diff
@@ -8,7 +8,7 @@ echo 'Extracting translatable strings from Django code.'
 (cd ${KOBOFORM_SRC_DIR} && python manage.py makemessages --locale en)
 
 echo 'Extracting translatable strings from client code.'
-(cd ${KOBOFORM_SRC_DIR} && python manage.py makemessages --locale en --domain djangojs --extension es6)
+(cd ${KOBOFORM_SRC_DIR} && python manage.py makemessages --locale en --domain djangojs --extension js)
```

Comment:
I'm unsure how this step relates to our extracted-strings.json generated from the webpack step.

But there is [i18MissingStrings.es6](https://github.com/kobotoolbox/kpi/pull/5463/files#diff-e2feb9bb7029efd7bf358c230532288e78c2094fd0f57275e07637212124fdd0) (now .js), so in case that's related, :+1: to this change.

Reply:
[💬 xref internal discussion](https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/django-admin.20makemessages/near/546678)